compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -1157,4 +1157,26 @@ getDefaultVscaleRange(IREE::HAL::ExecutableTargetAttr targetAttr) {
   return std::nullopt;
 }
 
+FailureOr<DimBoundSize>
+computeDimUpperBound(Value shapedValue, unsigned dimNum,
+                     std::optional<VscaleRange> vscaleRange) {
+  if (!vscaleRange.has_value()) {
+    FailureOr<int64_t> maybeDimBoundSize =
+        ValueBoundsConstraintSet::computeConstantBound(
+            presburger::BoundType::UB, {shapedValue, dimNum},
+            /*stopCondition=*/nullptr, /*closedUB=*/true);
+    if (succeeded(maybeDimBoundSize))
+      return DimBoundSize{.baseSize = *maybeDimBoundSize, .scalable = false};
```

Comment:
This broke the build on my MSVC/Windows machine: 
`[build] D:\dev\projects\iree\compiler\src\iree\compiler\Codegen\Utils\Utils.cpp(1169): error C7555: use of designated initializers requires at least '/std:c++20'`

Reply:
Here's a quick fix: https://github.com/iree-org/iree/pull/17968/